docs/Common/Structs/index.md

```diff
@@ -1179,3 +1182,40 @@ There are no defined parameters for this struct
 |:---|:---|:--------|:---------|:----------|
 |location|Common.Grid|true|||
 |state|Common.WindowState|true|||
+
+### DoorStatus
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|location|Common.Grid|true||Describes the status of a location of a door.|
```

Comment:
Please add reference links for Grid, DoorStatusType

Reply:
it makes no sense to add a link in Structs.index to another struct, it doesn't work on the portal.
as for, `DoorStatusType` - it is an enum, and there the link is provided, please see line 1191

`|status|[Common.DoorStatusType](../enums/#doorstatustype)|true||Describes the status of a door.|`

Reply:
All the link to enums file do not work any ways on master as well. It seems due to incorrect case used in links references(enums vs Enums). This needs to be corrected for all references to `Enums` file. e.g. above reference should change from:
```
|status|[Common.DoorStatusType](../enums/#doorstatustype)|true||Describes the status of a door.|
```

to

```
|status|[Common.DoorStatusType](../Enums/#doorstatustype)|true||Describes the status of a door.|
```

Reply:
as far as I can see, the rest of the reference links to enums in Struckts.index ([on develop](https://raw.githubusercontent.com/smartdevicelink/sdl_hmi_integration_guidelines/develop/docs/Common/Structs/index.md)) are written the same way, i.e not with capital E.
the magic starts when the PR is merged, only then the links start working.

Reply:
Yeah i checked that on master and it doesn't seem to work at least on my machine. 

Open: https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/master/docs/Common/Structs/index.md#didresult

and click on link to `Common.VehicleDataResultCode` ..  which opens up: https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/master/docs/Common/enums/#vehicledataresultcode and that is not working for me:

![image](https://user-images.githubusercontent.com/18143556/101919587-bd2f6600-3b98-11eb-94a0-40129495717e.png)


But if i change the link to: https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/master/docs/Common/Enums/index.md#vehicledataresultcode then it works:

![image](https://user-images.githubusercontent.com/18143556/101919841-0da6c380-3b99-11eb-95c5-906b353fcd48.png)

Reply:
@atiwari9, please see the links to enums updated per your request in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/232/commits/82d254e68a7968020ed1af7c00a92681e66c4c34